publiq-ui/pub-pagination.vue

```diff
@@ -3,10 +3,12 @@
     v-model="currentPage"
     :total-rows="rows"
```

Comment:
Only noticed this now, but maybe `total` would be better instead of `rows`? Or `total-rows`. Because `rows` can easily be misinterpreted as the amount of visible rows instead. I was confused myself at first when I saw the prop in Storybook.